cloud_GCP/terraform/module/examples/subnet-list/main.tf

```diff
@@ -1,16 +1,32 @@
 terraform {
   required_version = ">= 0.12.0"
+  required_providers {
+    google = {
+      version = ">= 3.41.0"
+    }
+    kentik-cloudexport = {
+      version = "0.1.0"
+      source = "kentik/kentik-cloudexport"
+    }
+  }
 }
 
 provider "google" {
-  version = ">= 3.41.0"
-  credentials = file("${var.credentials}")
+  credentials = file(var.credentials)
   project     = var.project
   region      = var.region
 }
 
+provider "kentik-cloudexport" {
+
```

Comment:
maybe it's worthwhile to add comment to cloudexport providers that email and token are read from env variables, for clarity?

Reply:
Resolved